tests/utils.py

```diff
@@ -30,3 +34,26 @@ def page_displays_error_matching_message(response, message: str) -> bool:
     )
 
     return True if error_summary else False
+
+
+def multidict_from_measure_version_with_overrides(measure_version: MeasureVersion, **kwargs) -> Dict[str, str]:
```

Comment:
I find the `with_overrides` a bit of an odd name.  It just means in the sense that you can specify kwargs that will override what was in your passed-in measure? 

I don't have any better suggestions right now though, and it needs something other than just `multidict_from_measure_version`.

Reply:
I'll change it to `multidict_from_measure_version_and_kwargs`